web/src/admin.js

```diff
@@ -70,6 +70,9 @@ const admin_settings_label = {
     realm_enable_guest_user_indicator: $t({
         defaultMessage: "Display “(guest)” after names of guest users",
     }),
+    realm_enable_guest_user_notification: $t({
+        defaultMessage: "Notify when a DM has a guest recipient",
```

Comment:
Let's discuss in the design thread, but I think "Display a warning when composing a direct message with guest user recipients."